src/test/java/no/difi/statistics/CategoriesQueryTest.java

```diff
@@ -0,0 +1,167 @@
+package no.difi.statistics;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.difi.statistics.elasticsearch.commands.CategoriesQuery;
+import no.difi.statistics.model.OwnerCategories;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CategoriesQueryTest {
+
+    @BeforeAll
+    static void setup() {}
+
+    @BeforeEach
+    void init() {}
+
+    @Test
+    public void testAddition() {
+        assertEquals(2, 1 + 1);
+    }
+
+    @Test
+    public void testMinuteDistance() {
+        CategoriesQuery categoriesQuery = new CategoriesQuery();
+        String distanceToken = "minute2022";
+        String distance = categoriesQuery.determineDistance(distanceToken);
+        assertEquals("minutes", distance);
+    }
+
+    @Test
+    public void testHourDistance() {
+        CategoriesQuery categoriesQuery = new CategoriesQuery();
+        String distanceToken = "hour2022";
+        String distance = categoriesQuery.determineDistance(distanceToken);
+        assertEquals("hours", distance);
+    }
+
+    @Test
+    public void testKeyIsPresent() {
+        Path path = Paths.get("src/test/java/no/difi/statistics/CategoriesMappingInput.json");
+        String mappings = null;
+        try {
+            mappings = Files.lines(path).reduce("", String::concat);
+        } catch (IOException e) {
```

Comment:
Istedet for å gjøre en try-catch og printe stacktrace i catchen, så legg heller på en throws i metode-signaturen. Så slipper du å håndtere feilene.